winpcapy.py

```diff
@@ -140,3 +140,8 @@ def capture_on_and_print(cls, pattern):
         Usage example capture_on_and_print("*Intel*Ethernet") will capture and print packets from an Intel Ethernet device
         """
         cls.capture_on(pattern, cls.packet_printer_callback)
+
+    @staticmethod
+    def capture_on_dev_name(device_name, callback):
```

Comment:
Huh, Yes I see your point now.
Of course if you want to capture on a device-guid/device-name directly you can simply call WinPcap() -e.g. - WinPcap('\\Device\\NPF_{0A78B7C8-F023-1337-1337-84D448AA5126}'), so I'd hardly call this an issue. But the utility function you suggest has its merits- so sure, why not.
Do you want to reopen a pull request with this addition (please name the function 'capture_on_device_name') , or would you rather I add it?